docs/joss_paper/paper.md

```diff
@@ -67,12 +67,12 @@ Unsupervised learning was first developed independently from a number of researc
 ![Registration network with unsupervised loss.\label{fig:unsupervised}](../asset/deepreg-tutorial-unsupervised.svg)
 
 ## Weakly-supervised learning
-Weak supervision utilising segmented corresponding regions in the medical image pairs was first proposed in a multimodal application for registering prostate MR to ultrasound images [@hu2018label]. In addition to the regularisation on the predicted displacement field, the training is driven by minimising the dissimilarity between the fixed labels and the warped moving labels, as illustrated in Fig.X. This formulation is modality-independent and similar to many other well-studied computer vision and medical imaging tasks, such as image segmentation. When multiple corresponding label pairs available, a two-stage sampling strategy can be adopted for an unbiased gradient back-propagation during training. Label dissimilarity measures include Dice, Jaccard, cross-entropy and their variants, such as a multiscale Dice.
+Weak supervision utilising segmented corresponding regions in the medical image pairs was first proposed in a multimodal application for registering prostate MR to ultrasound images [@hu2018label]. In addition to the regularisation on the predicted displacement field, the training is driven by minimising the dissimilarity between the fixed labels and the warped moving labels, as illustrated in Fig.X. This formulation is modality-independent and the label dissimilarity meausres are adapted from well-studied computer vision and medical imaging tasks, such as image segmentation. When multiple corresponding label pairs available, a two-stage sampling strategy can be adopted for an unbiased gradient back-propagation during training. Label dissimilarity measures include Dice, Jaccard, cross-entropy and their variants. DeepReg uses as a default multiscale Dice [@hu2018weakly].
 
 ![Registration network with weak supervision loss.\label{fig:weakly}](../asset/deepreg-tutorial-weakly.svg)
```

Comment:
Link broken, see above.

Reply:
"Weak supervision - which utilises.... - was first proposed."

Reply:
"as illustrated in Fig.X." Fig x?

Reply:
"label dissimilarity meausres" typo

Reply:
"When multiple corresponding label pairs available" ->  When multiple corresponding label pairs are available

Reply:
"can be adopted for an **unbiased** gradient back-propagation during training" what does this mean?

Reply:
"DeepReg uses as a default multiscale Dice [@hu2018weakly]." I would add: " - other dissimilarity measures are implemented for the user to tailor their training options" to highlight not only dice has been implemented.